tests/mocha/run_mocha_tests_in_browser.js

```diff
@@ -29,7 +29,12 @@ async function runMochaTestsInBrowser() {
   // Run in headless mode on Github Actions.
   if (process.env.CI) {
     options.capabilities['goog:chromeOptions'] = {
-      args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage']
+      args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage',
+        '--allow-file-access-from-files']
```

Comment:
Nit: [styleguide says continuation line should be +4 spaces](https://google.github.io/styleguide/jsguide.html#formatting-indent).

Reply:
Updated indentation, treating the array as a block , according to this rule: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation